packages/circle-react-elements/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,50 @@
+import { ArrowUpRight, CheckCircle } from 'lucide-react';
+
+import { Button } from '~/components/ui/button';
+
+export interface NotificationProps {
+  /** The wallet */
+  onClose?: () => void;
+  description: string | React.ReactElement;
+  externalLink?: string;
+  title: string;
+}
+
+/**
+ * Helpers for obtaining a wallet's on-chain address:
+ * a QR code that encodes the address and elements for viewing the address and copying it to the clipboard
+ */
+export function Notification({
```

Comment:
the comments in this file are wrong, please update, actually it's a good time to write a proper docs for this component too.
also, I think Notification is not a good name, this is more like SuccessMessage or ConfirmationMessage